db/seeds.rb

```diff
@@ -1,7 +1,17 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+Category.create!([
+  {name: 'Lenses'},
+  {name: 'Bodies'},
+  {name: 'Accessories'}
+  ])
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.